### ISSUE TITLE
File organization

### ISSUE THREAD

**Issue by SylvainTakerkart** [INT-NIT/AnDO#70]
Hi!

I think that the existence of the "tools" subdirectory is not necessary... I believe all components should be equal, i.e the checker, the generator (and maybe the viewer if we keep it).

Would it be possible to re-organize the files to get rid of the tools subdirectory and simply have:
ando/AndoChecker.py
ando/AndoGeneratory.py

@Slowblitz @JuliaSprenger , WDYT?

**Comment by JuliaSprenger** [comment]
Yes, with the current size the tools have this is a good idea. I would wait a few more weeks and see it any of them grow unexpectedly and then have another round of reformatting.
